tasks/process_agent.py

```diff
@@ -160,6 +160,6 @@ def __enter__(self):
         print("created tempdir: {name}".format(name=self.fname))
         return self.fname
 
-    def __exit__(self):
+    def __exit__(self, _, _, _):
```

Comment:
```suggestion
    def __exit__(self, _exception_type, _exception_value, _exception_traceback):
```

Reply:
Thanks @mx-psi, will check and update the PR once the Gitlab incident is resolved.